client/api/disk/v1alpha1/api.pb.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// source: github.com/kubernetes-csi/csi-proxy/client/api/disk/v1alpha1/api.proto
```

Comment:
Any ideas why the path is going to back to relative path in this case. Was the generate run with the latest make file (the latest code had the changes to make it absolute path) ?